src/core/agent/records/notificationRecord.ts

```diff
@@ -11,7 +11,7 @@ interface NotificationRecordStorageProps {
   read: boolean;
   multisigId?: string;
   connectionId: string;
-  linkedGroupRequests?: Record<string, boolean>;
+  linkedGroupRequests: Record<string, boolean>;
```

Comment:
Hmm, so this is `NotificationRecordStorageProps`, which is what we pass when creating the record. So it should be `linkedGroupRequests?` _here_, and when if pass undefined, it will fallback to `{}` in the constructor which is why the attribute is `linkedGroupRequests!` - but the props should be `?` right?